src/core/reactor.cc

```diff
@@ -95,7 +95,11 @@ module;
 #include <sys/mman.h>
 #include <sys/utsname.h>
 #include <linux/falloc.h>
+#ifdef SEASTAR_HAVE_SYSTEMTAP_SDT
 #include <sys/sdt.h>
```

Comment:
Maybe it's enough to use __has_include.

Reply:
@avikivity  that'd work also. actually i considered it. but again, i'd prefer searching non-standard library using `find_path()`. if you insist, i will use `__has_include`. could please let me know that you think.

Reply:
ok, if we use cmake above then we might as well use it here.